commcare_connect/opportunity/views.py

```diff
@@ -2591,6 +2597,67 @@ def add_api_key(request, org_slug):
     return HttpResponse(render_crispy_form(form))
 
 
+@require_POST
+@opportunity_required
+@org_member_required
+def invoice_items(request, *args, **kwargs):
+    body = json.loads(request.body)
+    start_date_str = body.get("start_date", None)
+    end_date_str = body.get("end_date", None)
+
+    if not start_date_str or not end_date_str:
+        return JsonResponse({"error": _("Start date and end date are required.")})
+
+    start_date = datetime.datetime.strptime(start_date_str, "%Y-%m-%d").date()
+    end_date = datetime.datetime.strptime(end_date_str, "%Y-%m-%d").date()
+
+    preview_size = 25
+    line_items, total_items_count = get_uninvoiced_visit_items(
+        request.opportunity, start_date, end_date, limit=preview_size
+    )
+    total_local_amount = sum(item["total_amount_local"] for item in line_items)
```

Comment:
I am getting pretty confused here but does this mean the totals are wrong if there are more than 25 items? Since we get the totals from the preview query.